src/pages/taxonomy/create.tsx

```diff
@@ -5,7 +5,7 @@ import { axGetTaxonRanks } from "@services/taxonomy.service";
 import React from "react";
 
 export default function TaxonomyCreatePage({ taxonRanksMeta }) {
-  return <SpeciesCreatePageComponent taxonRanksMeta={taxonRanksMeta} isSpeciesPage={false} />;
+  return <SpeciesCreatePageComponent taxonRanksMeta={taxonRanksMeta} isSpeciesPage={false} name={undefined}/>;
```

Comment:
instead of passing and processing undefined like this here `name={undefined}` , is there a better way to do it? can you look it up once?